#Params are in m and radians
/**:
  ros__parameters:
    path_waypoint_distance_tolerance: 4.0
    distance_tolerance: 1.0
    obj_distance_tolerance: 1.5 #TODO: Tune
    abort_distance_tolerance: 5.0
    aruco_distance_tolerance: 2.0
    hex_search_radius: 10.0
    navigate_speed: 10.0
    object_speed: 3.0
    aruco_speed: 3.0
    spin_speed: 20.0
    object_alpha_lpf: 0.5
    obj_enable_distance: 25.0
    aruco_enable_distance: 30.0
    aruco_alpha_lpf: 0.5
    spin_step_size: 0.6981
    spin_delay_time: 2.0
    wrong_aruco_backup_distance: 7.0
    wrong_aruco_backup_spin_speed: 15.0
    hex_seach_angle_difference: 50.0
    waypoint_distance: 10.0
    
    elevation_cost: 100.0 # How much elevation affects the cost of the computed AStar path
    # The below distance in meters creates angle with an 1 meter AStar step on the xy plane
    # For example, if you set the distance to 0.7, the maximum traversable angle is tan^-1(0.7/1.0) = 35 degrees
    elevation_limit: 0.35 # The maximum allowable elevation difference in the AStar planner to be considered traversable

fiducial_data:
  ros__parameters:
    simulation: false     #Run in simulation mode
    target_ar_tag_id: -1  #Target AR Tag ID